assets/stylesheets/modules/_promotion.scss

```diff
@@ -1,114 +1,222 @@
-    background-color: rgb(146, 202, 224);
+    background-color: guss-colour(news-support-1, $pasteup-palette);
 
     .page-slice {
         background-color: guss-colour(news-support-1, $pasteup-palette);
     }
 }
 
 .section-grey {
-    background-color: rgb(231, 231, 231);
+    background-color: guss-colour(neutral-7, $pasteup-palette);
 
     .page-slice {
-        background-color: rgb(241, 241, 241);
+        background-color: guss-colour(neutral-7, $pasteup-palette);
+    }
+}
+
+.section--dark-sides {
+    position: relative;
+
+    &:before {
+        background-color: rgba(51, 51, 51, 0.05);
+        content: '';
+        height: 100%;
+        position: absolute;
+        width: 100%;
     }
 }
 
 .digipack {
+    max-width: gs-span(14);
+    margin: 0 auto;
+    overflow: visible;
+}
 
+.digipack__roundel {
+    width: 100%;
 }
 
 .digipack__shots {
-    background: url(/assets/images/backgrounds/bg-promo-digital-pack.jpg) no-repeat center;
-    background-size: contain;
-    min-height: 135px;
-    min-width: 280px;
+    max-width: 780px;
 
-    @include mq(mobileLandscape) {
-        min-height: 211px;
-        min-width: 440px;
-    }
-    @include mq($from: phablet, $until: tablet) {
-        min-height: 286px;
-        min-width: 595px;
-    }
-    @include mq(desktop) {
-        min-height: 286px;
-        min-width: 595px;
-    }
-    @include mq(leftCol) {
-        min-height: 336px;
-        min-width: 700px;
+    @include mq(tablet) {
+        float: left;
+        padding-top: 0;
+        margin: 0 ($gs-baseline * 2) $gs-baseline 0;
+        width: 65%;
     }
-    @include mq(wide) {
-        min-height: 374px;
-        min-width: 780px;
+
+    img {
+        padding-top: 60px;
+        @include mq(tablet) {
+            padding-top: $gs-baseline;
+        }
     }
 }
 
 .digipack__benefits {
-    @include mq($until: tablet) {
-        width: 100% !important;
-    }
+    padding-top: $gs-baseline * 2;
 }
 
 .digipack__benefit {
-    border-color: $c-neutral5;
-    border-style: solid;
+    border-color: $c-neutral3;
+    border-style: dotted;
     border-width: 1px 0 0 0;
-    font-size: larger;
-    margin: 0;
-    padding: ($gs-baseline * 1.33) 0;
+    margin-bottom: $gs-baseline;
+    padding: ($gs-baseline / 3) 0;
 }
 
-.promotion-image {
+.discover {
 
 }
 
-.promotion-description {
-    @include mq(tablet) {
-        font-size: larger;
+.discover__screenshots {
+    padding: 18px 0 22px 0;
+}
+
+.discover__screenshots--in-grid {
+    padding: 0;
+}
+
+.discover__item {
+    position: relative;
+}
+
+.discover__item__icon {
+    position: absolute;
+
+    @include mq(phablet) {
+        position: relative;
     }
 }
 
-.promotion-cta {
+.discover__item__text {
+    border-color: $c-neutral3;
+    border-style: dotted;
+    border-width: 1px 0 0 0;
+    margin-left: 70px;
+    padding-top: ($gs-baseline / 3);
+
+    @include mq(phablet) {
+        border: 0;
+        margin-left: 0;
+    }
+
+    > h3 {
```

Comment:
I think the BEM convention is to avoid path based selectors like this in favour of more explicit class names